src/redprl/syntax.sml

```diff
@@ -16,14 +16,18 @@ struct
      VAR of variable * sort
    (* the trivial realizer for equality, which is called 'axiom' in NuPRL. *)
    | AX
+   (* the trivial realizer for interalized equality, less than, etc. *)
```

Comment:
It might be better to distinguish these two 'trivial' realizers by saying that one targets forms of judgment which have no computational content, and the other targets types whose elements have no computational content.